includes/ECommerce.php

```diff
@@ -143,6 +146,87 @@ public function register_settings() {
 				'description'  => __( 'NFD eCommerce Options', 'wp-module-ecommerce' ),
 			)
 		);
+		\register_setting(
```

Comment:
@sumathi3399  Can you try combinining these into one `register_settings` to avoid code duplication?
Refer to this one:
https://stackoverflow.com/questions/29795703/passing-array-as-option-value-in-wordpress-function-register-setting

Reply:
done